x/evm/keeper/permissions.go

```diff
@@ -0,0 +1,92 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package keeper
+
+import (
+	"slices"
+
+	"github.com/evmos/evmos/v18/x/evm/types"
+)
+
+type PermissionPolicy interface {
```

Comment:
interfaces shouldn't be in the keeper otherwise it is likely to cause circular dependencies. Let's move this to types

Reply:
and please don't forget the comments